### PR TITLE
Adding Footer

### PR DESCRIPTION
**Pull request by mikefranze**
Adds basic footer, recommendations for what else to put are welcome. 
Some potentials would be site navigation, GitHub contribution info, copyright info, feedback options.

Also changes STAR Voting Elections Hosting to STAR Elections, will likely change again later.

image
image

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -39,16 +40,24 @@ const App = () => {
...
-            <Route path='/Debug' element={<DebugPage authSession={authSession} />} />
-            <Route path='/CreateElection' element={<AddElection authSession={authSession} />} />
-            <Route path='/Election/:id/*' element={<Election authSession={authSession} />} />
-            <Route path='/DuplicateElection/:id' element={<DuplicateElection authSession={authSession} />} />
-            <Route path='/Sandbox' element={<Sandbox />} />
-          </Routes>
+          <Box
+            sx={{
+              width: '100%',
+              minHeight: `calc(100vh - 255px - 64px)` //Sets min height of content to 100% minus footer and header in order to force footer to the bottom
```
Comment (ArendPeter) [inline frontend/src/App.tsx:46]: This height logic will probably break with the new responsive header that can take multiple rows

Maybe you can put the header, middle, and footer all in a flex box and set the middle to flex grow?

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -39,16 +40,24 @@ const App = () => {
...
+              <Route path='/Login' element={<Login />} />
+              <Route path='/Debug' element={<DebugPage authSession={authSession} />} />
+              <Route path='/CreateElection' element={<AddElection authSession={authSession} />} />
+              <Route path='/Election/:id/*' element={<Election authSession={authSession} />} />
+              <Route path='/DuplicateElection/:id' element={<DuplicateElection authSession={authSession} />} />
+              <Route path='/Sandbox' element={<Sandbox />} />
+            </Routes>
+
+          </Box>
+          <Footer />
```
Comment (ArendPeter) [inline frontend/src/App.tsx:60]: I was thinking about if we wanted to do a call to action for contributions somewhere in the footer, but I don't think we need one. I elaborated on my thoughts [here](https://github.com/Equal-Vote/star-server/issues/289)

**Review by ArendPeter — CHANGES_REQUESTED** [review]
Nice work! Just left a comment about the minHeight calculation

File: frontend/src/App.tsx (hunk 1 above)
Comment (mikefranze) [reply frontend/src/App.tsx]: Done, but by setting footer margin top to auto.

**Pull request merged**